Dockerfile

```diff
@@ -21,10 +21,18 @@ RUN apk update && apk add --no-cache \
   python3 \
   procps \
   coreutils \
-  mongodb-tools \
   postgresql15-client \
   redis \
-  nginx
+  nginx && \
+  # Conditionally install mongodb-tools if not s390x
+  if [ "$TARGETOS" = "linux" ] && [ "$TARGETARCH" != "s390x" ]; then \
+    apk add --no-cache mongodb-tools; \
+  fi && \
+  # Conditionally install oha if not s390x
+  if [ "$TARGETOS" = "linux" ] && [ "$TARGETARCH" != "s390x" ]; then \
+    wget -qO /usr/local/bin/oha https://github.com/hatoo/oha/releases/latest/download/oha-linux-${TARGETARCH} && \
+    chmod +x /usr/local/bin/oha; \
+  fi
```

Comment:
@rajnish-jauhari , there is some duplication here, and also the wget on the same RUN as the apk looks a bit weird.  

What do you think moving both Z-specific commands to their own RUN?  Something like this:

```
RUN  if [ "$TARGETOS" = "linux" ] && [ "$TARGETARCH" != "s390x" ]; then \
    apk add --no-cache mongodb-tools && \
    wget -qO /usr/local/bin/oha https://github.com/hatoo/oha/releases/latest/download/oha-linux-${TARGETARCH} && \
    chmod +x /usr/local/bin/oha; \
  fi
```

We end up with two layers instead of just one, but I think the Dockerfile would me more readable that way.